bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/AuditorLedgerCheckerTest.java

```diff
@@ -767,6 +772,16 @@ public void testDelayedAuditWithRollingUpgrade() throws Exception {
         LOG.info("*****************Test Complete");
     }
 
+    private void waitForAuditToComplete() throws Exception {
```

Comment:
If there is no Auditor or no AuditTask should we fail the test ?
Or shall we wait for an Auditor to be active ?

Reply:
Yes that is potentially a real concern.  I will add it in.  In practice it should not be an issue, because the auditor is started up but the setup of this test class, but we probably want to be sure of it.